packages/components/inputs/rich-text-utils/src/localized/localized.ts

```diff
@@ -0,0 +1,66 @@
+import { warning } from '@commercetools-uikit/utils';
+import uniq from 'lodash/uniq';
+import html from '../html';
+import isRichTextEmpty from '../is-empty';
+
+type TOmitEmptyTranslations = {
+  language: string;
+};
+
+type TCreateLocalizedString = {
+  [key: string]: string;
+};
+
+const initializeValue = (value: string) =>
+  html.serialize(html.deserialize(value));
+
+const isLocalizedHtmlValueEmpty = (value: unknown) =>
+  !value || isRichTextEmpty(value);
+
+export const isEmpty = (
+  localizedHtmlValue:
+    | { [key: string]: unknown }
+    | ArrayLike<unknown>
+    | undefined
+) => {
+  if (!localizedHtmlValue) return true;
+  return Object.values(localizedHtmlValue).every(isLocalizedHtmlValueEmpty);
+};
+
+export const omitEmptyTranslations = (
+  localizedString: TOmitEmptyTranslations
+) => {
+  warning(
+    typeof localizedString === 'object',
+    'omitEmptyTranslations must be called with an object'
+  );
+  return Object.entries(localizedString).reduce<Record<string, unknown>>(
+    (localizedStringWithoutEmptyTranslations, [language, value]) => {
+      if (!isLocalizedHtmlValueEmpty(value)) {
+        localizedStringWithoutEmptyTranslations[language] = value;
+      }
+      return localizedStringWithoutEmptyTranslations;
+    },
+    {}
+  );
+};
+
+export const createLocalizedString = (
+  languages: string[],
+  existingTranslations = {} as TCreateLocalizedString
```

Comment:
As the type is typing a single parameter I was expecting it to have a name resembling the expected value; something like `TTranslations`.
What do you think?